### PR TITLE
Feature/gapple

### PR DESCRIPTION
**Pull request by ubamrein**
Adds initial support for google/apple approach

### CODE REVIEW

File: dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/DPPPTController.java [hunk 1]
```diff
@@ -124,6 +125,44 @@ public DPPPTController(DPPPTDataService dataService, EtagGeneratorInterface etag
...
+			exposee.setKey(exposedKey.getKey());
+			long keyDate = this.validateRequest.getKeyDate(principal, exposedKey);
+
+			exposee.setKeyDate(keyDate);
+			if(!this.validateRequest.isFakeRequest(principal, exposeeRequests)) {
+				
+				exposees.add(exposee);
+			} 
+		}
+		dataService.upsertExposees(exposees, appSource);
```
Comment (AndreasHuber-CH) [inline dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/DPPPTController.java:153]: Fake requests trigger an SQL batch update with an empty list here.  Would it make sense to invert the for loop and the if statement? or skipping the SQL query when the list is empty?

Comment (ubamrein) [reply dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/DPPPTController.java]: yes you're right of course :D Forgot to take it out of the loop. Now it should only upsert, if it is not a fake request

File: dpppt-backend-sdk/dpppt-backend-sdk-model/src/main/java/org/dpppt/backend/sdk/model/ExposeeRequestList.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package org.dpppt.backend.sdk.model;
+
+import java.util.List;
+
+public class ExposeeRequestList {
+    List<ExposedKey> exposedKeys;
```
Comment (bachmann-ubique) [inline dpppt-backend-sdk/dpppt-backend-sdk-model/src/main/java/org/dpppt/backend/sdk/model/ExposeeRequestList.java:6]: müsste die nicht @NotNull sein?

Comment (bachmann-ubique) [reply dpppt-backend-sdk/dpppt-backend-sdk-model/src/main/java/org/dpppt/backend/sdk/model/ExposeeRequestList.java:6]: und @NotEmpty

**Pull request merged**